### PR TITLE
659 improve mongodb docs

### PR DESCRIPTION
**Pull request by xtfer**
Improve the MongoDB docs for #659 

- Fix an issue with connection strings
- Add a NodeJS example
- Minor wording changes

### CODE REVIEW

File: src/configuration/services/mongodb.md [hunk 1]
```diff
@@ -62,16 +62,33 @@ foreach ($relationships['database'] as $endpoint) {
   if (empty($endpoint['query']['is_master'])) {
     continue;
   }
-  $container->setParameter('mongodb_server', $endpoint['scheme'] . '://' . $endpoint['host'] . ':' . $endpoint['port']);
+  $container->setParameter('mongodb_server', $endpoint['scheme'] . '://' . $endpoint['ip'] . ':' . $endpoint['port']);
```
Comment (mrkschan) [inline src/configuration/services/mongodb.md]: why use the `ip`?

Comment (xtfer) [reply src/configuration/services/mongodb.md]: 'host' appears not to work in some configurations, whereas IP is always available, AFAIK.

File: src/configuration/services/mongodb.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
 # MongoDB (Database service)
 
-MongoDB is a cross-platform document-oriented database.
-
-See the [MongoDB documentation](https://docs.mongodb.com/manual/) for more information.
+MongoDB is a cross-platform, document-oriented database.
```
[the remark this answers is not in the document]
Comment (xtfer) [reply src/configuration/services/mongodb.md]: Its not gone, I moved it to the bottom of the page, as the existing phrasing suggested that the link was part of our documentation.

[the remark this answers is not in the document]
Comment (Crell) [reply src/configuration/services/mongodb.md]: That makes it inconsistent with the rest of the service pages.  The others all have links in the opening section.  We should be consistent.  Changing that across all services would be something to discuss elsewhere.

File: src/configuration/services/mongodb.md [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 
 ## Supported versions
 
 * 3.0
 
-## Relationship
+## Relationships
```
Comment (Crell) [inline src/configuration/services/mongodb.md]: It's `Relationship` in every other service page.  Again, consistency.  Changing the pattern can be discussed elsewhere.

**Pull request merged**